common/Environments/Scripts/HyperVSetupScript.cs

```diff
@@ -37,21 +37,10 @@ 6. The user is already in the Hyper-V Admin group and the Hyper-V Feature is alr
             $adminGroupResult = [OperationStatus]::OperationNotRun
             $hyperVGroupSid = 'S-1-5-32-578'
 
-            # Check the security token the user logged on with contains the Hyper-V Administrators group SID (S-1-5-32-578). This can only be updated,
-            # once the user logs off and on again. Even if we add the user to the group later on in the script.
```

Comment:
Not understanding the suggestion? And what's the advantage?

Reply:
Oh this was a suggestion sorry. It was for if you wanted to pass in the users S-I-D of the user since you removed lines 42. which had

`[System.Security.Principal.WindowsIdentity]::GetCurrent().Groups.Value`

I was thinking there might be a way to pass in something like a SID or username so we can get an object that would tell us if the user is in the group or not. But if we really don't need these lines then its all good to ignore.